src/Components/DishCard/DishCard.js

```diff
@@ -8,17 +8,36 @@ import Loading from '../Loading/Loading';
 const DishCard = ({ dishId, name, rating, getRestaurant, reviews, user, restaurantId }) => {
 
   const printPeppers = () => {
-    if (rating === 1) {
-      return <>🌶</>
-    } else if (rating === 2) {
-      return <>🌶🌶</>
-    } else if (rating === 3) {
-      return <>🌶🌶🌶</>
-    } else if (rating === 4) {
-      return <>🌶🌶🌶🌶</>
-    }
-    else if (rating === 5) {
-      return <>🌶🌶🌶🌶🌶</>
+    console.log(rating)
+    let spiceRatingIndex;
+    if(reviews.length > 0) {
+      spiceRatingIndex = Math.floor(Math.random() * reviews.length);
+      console.log(reviews[spiceRatingIndex].overallRating)
+      if (reviews[spiceRatingIndex].overallRating === 1) {
+        return <>🌶</>
+      } else if (reviews[spiceRatingIndex].overallRating === 2) {
+        return <>🌶🌶</>
+      } else if (reviews[spiceRatingIndex].overallRating === 3) {
+        return <>🌶🌶🌶</>
+      } else if (reviews[spiceRatingIndex].overallRating === 4) {
```

Comment:
This is both really good/impressive code and a good argument to make backend do this averaging functionality if we continue to work on this project after code freeze.